### PR TITLE
Verify button

### LINKED ISSUE #113: Verify button in each content
final stretch!
add a button which is only accessible for admins, which would send a `PUT` request to the otp to update a content

### PR DESCRIPTION
**Pull request by Karyum**
close #113 

the reason this PR is a bit big is becasue i had to change both events and places verify and found bit of bugs on the way

### CODE REVIEW

**Comment by des-des** [comment]
@Karyum Everything is `verify.js` is in global scope

File: public/verify.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+var verifyButton = document.getElementById('verify-button');
```
Comment (des-des) [inline public/verify.js]: This is in global scope !! ?? :cry:

File: public/verify.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+var errorContainer = document.getElementById('error-container');
+var lang = window.location.pathname.split('/')[1];
+var resourceType = window.location.pathname.split('/')[2];
+var id = window.location.pathname.split('/')[3];
```
Comment (des-des) [inline public/verify.js]: `id` is a very bad name (especially for something in global scope)

File: public/verify.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+var url = 'http://localhost:3000/api/v1/' + resourceType;
```
Comment (des-des) [inline public/verify.js]: ? Localhost?

File: public/verify.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+verifyButton.addEventListener('click', function() {
+  axios
+    .get(url + 's/' + id)
+    .then(function(res) {
+      var verifiedResource = Object.assign({}, res.data, {
```
Comment (des-des) [inline public/verify.js]: Browser comparability?

**Review by des-des — CHANGES_REQUESTED** [review]

**Comment by Karyum** [comment]
yeah sorted it out :sweat_smile: :man_facepalming:

**Review by des-des — APPROVED** [review]

**Pull request merged**